bech32/src/Codec/Binary/Bech32/Internal.hs

```diff
@@ -372,6 +376,52 @@ data EncodingError =
     -- 'encodedStringMaxLength'.
     deriving (Eq, Show)
 
+-- | Replace the encoded Bech32 string with new human-readable prefix.
+--
+-- == Example
+--
+-- >>> import Prelude
+-- >>> import Codec.Binary.Bech32
+-- >>> import Data.Text.Encoding
+--
+-- First, prepare a human-readable prefix:
+--
+-- >>> Right prefix = humanReadablePartFromText "example"
+--
+-- Next, prepare a data payload:
+--
+-- >>> messageToEncode = "Lorem ipsum dolor sit amet!"
+-- >>> dataPart = dataPartFromBytes $ encodeUtf8 messageToEncode
+--
+-- Then, produce a Bech32 string:
+--
+-- >>> Right bech32txt = encode prefix dataPart
+-- >>> bech32txt
+-- "example1f3hhyetdyp5hqum4d5sxgmmvdaezqumfwssxzmt9wsss9un3cx"
+--
+-- Finally, prepare a new human-readable prefix and replace bech32txt with it:
+--
+-- >>> Right newprefix = humanReadablePartFromText "newexample"
+-- >>> Right bech32txt' = replaceHumanReadablePart bech32txt newprefix
+-- >>> bech32txt'
+-- "newexample1f3hhyetdyp5hqum4d5sxgmmvdaezqumfwssxzmt9wsss3nvvwp"
+replaceHumanReadablePart :: Text -> HumanReadablePart -> Either ReplaceHrpError Text
```

Comment:
**Request**: wrap this to meet the [line length limit](https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#limit-line-length-to-80-characters).

```suggestion
replaceHumanReadablePart
    :: Text
    -> HumanReadablePart
    -> Either ReplaceHrpError Text
```